src/code42cli/cmds/trustedactivities.py

```diff
@@ -0,0 +1,191 @@
+import click
+from py42.clients.trustedactivities import TrustedActivityType
+
+from code42cli.bulk import generate_template_cmd_factory
+from code42cli.bulk import run_bulk_process
+from code42cli.click_ext.groups import OrderedGroup
+from code42cli.file_readers import read_csv_arg
+from code42cli.options import format_option
+from code42cli.options import sdk_options
+from code42cli.output_formats import OutputFormatter
+
+resource_id_arg = click.argument("resource-id", type=int)
+type_option = click.option(
+    "--type",
+    help="Type of trusted activity. `DOMAIN` or `SLACK`.",
+    type=click.Choice(TrustedActivityType.choices()),
+)
+value_option = click.option(
+    "--value",
+    help="The value of the trusted activity, such as the domain or Slack workspace name.",
+)
+description_option = click.option(
+    "--description", help="The description of the trusted activity."
+)
+
+
+def _get_trust_header():
+    return {
+        "resourceId": "Resource Id",
+        "type": "Type",
+        "value": "Value",
+        "description": "Description",
+        "updatedAt": "Last Update Time",
+        "updatedByUsername": "Last Updated By",
+    }
+
+
+@click.group(cls=OrderedGroup)
+@sdk_options(hidden=True)
+def trusted_activities(state):
+    """Manage trusted activities and resources."""
+    pass
+
+
+@trusted_activities.command()
+@click.argument("type", type=click.Choice(TrustedActivityType.choices()))
+@click.argument("value")
+@description_option
+@sdk_options()
+def create(state, type, value, description):
+    """Add a trusted activity.
+
+    VALUE is the name of the domain or Slack workspace.
+    """
+    state.sdk.trustedactivities.create(
+        type, value, description=description,
+    )
+
+
+@trusted_activities.command()
+@resource_id_arg
+@value_option
+@description_option
+@sdk_options()
+def update(state, resource_id, value, description):
+    """Update a trusted activity.  Requires the activity's resource ID."""
+    state.sdk.trustedactivities.update(
+        resource_id, value=value, description=description,
+    )
+
+
+@trusted_activities.command()
+@resource_id_arg
+@sdk_options()
+def remove(state, resource_id):
+    """Remove a trusted activity.  Requires the activity's resource ID."""
+    state.sdk.trustedactivities.delete(resource_id)
+
+
+@trusted_activities.command()
+@resource_id_arg
+@format_option
+@sdk_options()
+def show(state, resource_id, format):
```

Comment:
Oh thats a good point, I just included it because it was in the other commands.  Is this just one we use if the object contains more data fields than can reasonably be displayed in `list`?

Reply:
Yeah, `show` is for objects that have more data or need a more complex way to display it than just a table.